### PR TITLE
Cleanup of Guest History.

### PR DESCRIPTION
**Pull request by laxderek**
[#132011375]

https://www.pivotaltracker.com/story/show/132011375

### CODE REVIEW

**Comment by laxderek** [comment]
@eldavido PTAL

File: proto/booking/folio_charge_posted_notification.proto [hunk 1]
```diff
@@ -8,15 +8,13 @@ import "primitive/monetary_amount.proto";
...
 
 message FolioChargePostedNotification {
 	string j_w_token = 1;
 	holms.types.primitive.PbLocalDate opsdate = 2;
 	holms.types.primitive.MonetaryAmount pretax_lodging_charge = 3;
-	holms.types.operations.rooms.RoomIndicator room_id = 4;
-	holms.types.supply.room_types.RoomTypeIndicator room_type_id = 5;
-	holms.types.crm.guests.GuestIndicator guest_id = 6;
-	holms.types.booking.indicators.ReservationIndicator reservation_id = 7;
+	holms.types.supply.room_types.RoomTypeIndicator room_type_id = 4;
```
Comment (eldavido) [inline proto/booking/folio_charge_posted_notification.proto]: this looks good.

File: proto/crm/guests/guest_history.proto [hunk 2]
```diff
@@ -6,13 +6,13 @@ option go_package = "crm/guests";
 
 import "crm/guests/guest_indicator.proto";
 import "crm/guests/guest_cancelled_reservation.proto";
-import "crm/guests/guest_completed_stay.proto";
+import "crm/guests/guest_checked_in.proto";
 import "primitive/monetary_amount.proto";
 
 message GuestHistory {
```
Comment (eldavido) [inline proto/crm/guests/guest_history.proto]: You might want to call this, `RetrievedGuestHistory` to emphasize that it's something we're getting. Maybe `GuestHistorySummary` or `SummarizedGuestHistory`

**Comment by laxderek** [comment]
Closing in favor of newer branch.

**Pull request closed without merging**